.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+on: ["pull_request"]
+name: Github Action for proxy repository
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.9]
```

Comment:
this is quite specific, `1.13.x` is better
what about the other versions?

Reply:
@devimc ok let me do like 1.13.x , I am using 1.13 because I am following https://github.com/kata-containers/proxy/blob/master/.travis.yml#L22. Wdyt, should I test others?

Reply:
I think so, 1.14.x and 1.15.x